tests/integration/models/application.spec.coffee

```diff
@@ -142,14 +142,18 @@ describe 'Application Model', ->
 					promise = balena.models.application.hasAny()
 					m.chai.expect(promise).to.eventually.be.true
 
-			describe 'balena.models.application.create()', ->
+			describe.only 'balena.models.application.create()', ->
```

Comment:
I guess you forgot this one :)

Reply:
No, I added this on purpose on a separate commit with a `Change-type: dropme`, so that Akis can give it a try, since there seems to be a regression.